docs/advanced/sphinx.md

```diff
@@ -100,6 +100,42 @@ Simple is better than complex. 😵
 ```
 ````
 
+### Local Sphinx Extensions
+
+As detailed in its documentation, [Sphinx is able to use local extensions](https://www.sphinx-doc.org/en/master/development/tutorials/helloworld.html#using-the-extension) by adding additional directories to the [Python path](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH). At present, this functionality is not available within Jupyter Book.
+
+However, this limitation can be circumvented if one installs their Sphinx extension locally using `pip`. First, write a minimal `setup.py` as follows:
+
+```python
+from setuptools import setup
+
+setup(
+	name='my_sphinx_ext'
+)
+```
+
+For further documentation on the purposes and configuration of `setup.py`, please refer to [the official Python documentation on setup scripts](https://docs.python.org/3/distutils/setupscript.html).
```

Comment:
````suggestion
```{seealso}
For further documentation on the purposes and configuration of `setup.py`, please refer to [the official Python documentation on setup scripts](https://docs.python.org/3/distutils/setupscript.html).
```
````